web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -202,12 +203,24 @@
 
   let shiftKeyIsDown = false;
 
-  const onKeyDown = (event: KeyboardEvent) => {
+  const onKeyDown = async (event: KeyboardEvent) => {
     if ($isSearchEnabled) {
       return;
     }
 
-    if (matchesShortcut(event, { key: 'Shift', shift: true })) {
+    // Select All
+    if (matchesShortcut(event, { key: 'A', ctrl: true })) {
+      event.preventDefault();
+      await selectAllAssets(assetStore, assetInteractionStore);
+    }
+    // Deselect All
+    else if (matchesShortcut(event, { key: 'D', ctrl: true })) {
+      event.preventDefault();
+      $isSelectingAllAssets = false;
+      assetInteractionStore.clearMultiselect();
+    }
+    // Start Multi-Selection
+    else if (event.key === 'Shift') {
```

Comment:
What's the reason behind this change?

Reply:
Press Shift -> you enter in multi-selection mode
Then press Alt or Ctrl -> you are still in multi-selection mode (expected)
Then release Ctrl -> you are now stuck in multi-selection mode

Checking only the key code fixes that. I **know** there is a very low chance that someone does that, but that's still a (small) glitch waiting to happen

Reply:
Ah makes sense, didn't think about that